tests/python/pants_test/engine/test_build_files.py

```diff
@@ -123,10 +122,7 @@ def test_fails_on_nonexistent_specs(self) -> None:
         address_family = AddressFamily("root", {"a": ("root/BUILD", TargetAdaptor())})
         address_specs = AddressSpecs([SingleAddress("root", "b"), SingleAddress("root", "a")])
 
-        expected_rx_str = re.escape(
-            """"b" was not found in namespace "root". Did you mean one of:
-            :a"""
-        )
+        expected_rx_str = r'"b" was not found in namespace "root". Did you mean one of:\n\s+:a'
```

Comment:
I personally loved the string literal in the previous one for error message formatting specifically, but I can live with a regex.